packages/ckeditor5-dev-web-crawler/src/constants.ts

```diff
@@ -40,6 +40,7 @@ export const IGNORED_HOSTS = [
 	'challenges.cloudflare.com',
 	'binance.com',
 	'iframe.ly',
+	'if-cdn.com',
```

Comment:
```suggestion
	// See: https://github.com/ckeditor/ckeditor5/issues/18519.
	'if-cdn.com',
```

To know why. That's why I added the new entry at the end.